hudi-client/hudi-client-common/src/test/java/org/apache/hudi/testutils/GenericRecordValidationTestUtils.java

```diff
@@ -93,29 +92,31 @@ public static void assertGenericRecords(GenericRecord record1, GenericRecord rec
   }
 
   public static void assertDataInMORTable(HoodieWriteConfig config, String instant1, String instant2,
-                                          StorageConfiguration<Configuration> storageConf, List<String> partitionPaths) {
+                                          StorageConfiguration<?> storageConf, List<String> partitionPaths) {
     List<String> excludeFields = CollectionUtils.createImmutableList(COMMIT_TIME_METADATA_FIELD, COMMIT_SEQNO_METADATA_FIELD,
         FILENAME_METADATA_FIELD, OPERATION_METADATA_FIELD);
     assertDataInMORTable(config, instant1, instant2, storageConf, partitionPaths, excludeFields);
   }
 
   public static void assertDataInMORTable(HoodieWriteConfig config, String instant1, String instant2,
-                                          StorageConfiguration<Configuration> storageConf, List<String> partitionPaths, List<String> excludeFields) {
-    JobConf jobConf = new JobConf(storageConf.unwrap());
+                                          StorageConfiguration<?> storageConf, List<String> partitionPaths, List<String> excludeFields) {
+    StorageConfiguration<?> storageConfCopy = storageConf.newInstance();
```

Comment:
We might still keep the same logic as before as `HadoopFSUtils.convertToJobConf` is used to convert the config back to `JobConf`.